### DISCUSSION TITLE
Call for volunteer maintainers

### DISCUSSION THREAD

**Discussion by dhuseby** [libp2p/rust-libp2p#5154] [General]
Hello rust-libp2p!! 👋🏻

If you look at the github-mgmt repo at who the [maintainers of rust-libp2p are](https://github.com/libp2p/github-mgmt/blob/e1c4dfa3fc7591cb04fc469c9690de8c12e44da5/github/libp2p.yml#L8294), you'll see that the current list includes:
* @mxinden 
* @thomaseizinger 
* @jxs 
* @MarcoPolo 

As most of you know, @mxinden has decided to take time off after leaving Protocol Labs and @thomaseizinger is no longer on contract to be the day-to-day maintainer of rust-libp2p. @MarcoPolo is on leave as well but should be coming back pretty soon. That leaves us with @thomaseizinger and @jxs doing the bulk of the landing of PRs on a voluntary basis. I think it is time to discuss expanding the list of volunteer maintainers to help with the load of issues and PR's coming in. 

We have a rust-libp2p Open Maintainers Call (open to the whole community) coming up on February 13th. Here's the event page on lu.ma, I encourage all of you to register to that I can gauge interest: https://lu.ma/rust-libp2p  I put together an agenda doc here that is public, please feel free to add anything you'd like to dicuss: https://docs.google.com/document/d/1YezMvoDWcjFPb_wtgoFRQCSkvACg1cKwrB2vAzXsYQI/edit?usp=sharing

In the agenda, the first item is a discussion about maintainership. If you'd like to volunteer to help, please reply here and/or attend the maintainers call and I can help you get set up with the correct expectations and permissions.

Cheers!! 🍻

**Comment by thomaseizinger** [comment]
I can't join this because it is at 2am for me but here is my update:

- My main priority is merging AutoNATv2 to ensure @umgefahren can finish his contract and we haven't done that massive piece of work for nothing.
- It would be cool to also land WebTransport server support in some form or another. See https://github.com/libp2p/rust-libp2p/pull/4874. I am open to volunteering some time to this as well (cc @dgarus )
- Outside of that, I am not planning on spending unpaid time on `rust-libp2p` in the future.

**Reply by dhuseby** [reply]
Dang, we have one of those situations where there's people involved that are roughly 1/3 of the planet from each other. @thomaseizinger what if I move it to 14:00 UTC-7, which is 08:00 UTC+11 for you and 22:00 UTC+1 for our friends in Europe. That's pretty much the only window that is *sort of* OK for everybody.

**Reply by dhuseby** [reply]
@thomaseizinger I've added your updates to the notes for the meeting tomorrow.

**Reply by thomaseizinger** [reply]
> Dang, we have one of those situations where there's people involved that are roughly 1/3 of the planet from each other. @thomaseizinger what if I move it to 14:00 UTC-7, which is 08:00 UTC+11 for you and 22:00 UTC+1 for our friends in Europe. That's pretty much the only window that is _sort of_ OK for everybody.

No need to move it for me, I don't think I'd have much to add to the above :)

**Comment by emnul** [comment]
@dhuseby I'm currently getting up to speed on the project, but I'm certainly interested in working towards becoming a maintainer. I plan on attending the Jul 16 open maintainers call. See you then!
